core/io/file_access_pack.cpp

```diff
@@ -123,15 +123,17 @@ PackedData::~PackedData() {
 
 //////////////////////////////////////////////////////////////////
 
-bool PackedSourcePCK::try_open_pack(const String &p_path, bool p_replace_files) {
+bool PackedSourcePCK::try_open_pack(const String &p_path, bool p_replace_files, size_t p_offset) {
 	FileAccess *f = FileAccess::open(p_path, FileAccess::READ);
 	if (!f) {
 		return false;
 	}
 
+    f->seek(p_offset);
```

Comment:
Do we need to do a range check of any kind to make sure the offset is within the file?

Reply:
I double checked ```seek()``` implementation: [Android](https://github.com/godotengine/godot/blob/00949f0c5fcc6a4f8382a4a97d5591fd9ec380f8/platform/android/file_access_android.cpp#L75) [Unix](https://github.com/godotengine/godot/blob/00949f0c5fcc6a4f8382a4a97d5591fd9ec380f8/drivers/unix/file_access_unix.cpp#L187) [Windows](https://github.com/godotengine/godot/blob/00949f0c5fcc6a4f8382a4a97d5591fd9ec380f8/drivers/windows/file_access_windows.cpp#L192)
The EOF situation will be handled.